html/income.html

```diff
@@ -173,14 +181,27 @@
           return response.employments
         }
 
+        const createRefreshTask = async () => {
+          console.log("Requesting a data refresh from the quickstart back end.")
+          const response = await fetch(apiEnpoint + `createRefreshTask`, {
```

Comment:
```suggestion
          const response = await fetch(apiEndpoint + `createRefreshTask`, {
```

Plus 3-4 other uses like that